source/agora/common/Deserializer.d

```diff
@@ -184,6 +196,12 @@ public void deserializePart (ref ubyte record, scope DeserializeDg dg)
     record = dg(record.sizeof)[0];
 }
 
+/// Ditto
+public void deserializePart (ref bool record, scope DeserializeDg dg) @safe
```

Comment:
A bool type has been added to deserialize part, 
but it should also be implemented in serialize part.